src/System.Data.SqlClient/ref/Configurations.props

```diff
@@ -2,13 +2,11 @@
   <PropertyGroup>
     <PackageConfigurations>
       netcoreapp2.1;
-      netstandard2.0;
-      net461;
     </PackageConfigurations>
     <BuildConfigurations>
       $(PackageConfigurations)
       netcoreapp;
-      netfx;
+      _netfx;
```

Comment:
Not required since this doesn't build for `netstandard`: none of the remaining configurations would apply to netfx.